Library/Formula/slib.rb

```diff
@@ -0,0 +1,24 @@
+class Slib < Formula
+  homepage "http://people.csail.mit.edu/jaffer/SLIB"
+  url "http://groups.csail.mit.edu/mac/ftpdir/scm/slib-3b4.zip"
+  version "3b4"
+  sha1 "dda1ed78ff7164738a1a8c51f1f7c08ec1db79eb"
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "infoz", "INSTALL_INFO=install-info"
+    system "make", "install", "INSTALL_INFO=install-info"
+  end
+
+  def caveats; <<-EOS.undent
+    SLIB distribution is installed at /usr/local/lib/slib/.
+
+    See `info slib' to see how to configure your Scheme implementation to
+    locate SLIB, load SLIB initialization file and so on.
+    EOS
+  end
+
+  test do
+    system "#{bin}/slib", "-v"
```

Comment:
Thank you so much for your comment. Would it be possible for you to let me know any examples which depend on other application formulae? To do meaningful test on SLIB, we need to know which one of the following scheme implementations is installed on the system: MIT scheme, scm, gsi, gosh, guile, slib48, larceny, scmlit, elk, sisc, kawa or mzscheme.